packages/core/src/__tests__/data/blocks/builder-render-content.raw.tsx

```diff
@@ -0,0 +1,92 @@
+import { Show, onMount, useState, For, afterUnmount } from '@jsx-lite/core';
+import { Builder, BuilderContent, GetContentOptions } from '@builder.io/sdk';
+import { applyPatchWithMinimalMutationChain } from '@builder.io/utils';
+import { useBuilderData } from '@builder.io/jsx-lite';
+import { RenderBlock } from './builder-render-block.raw';
+
+type RenderContentProps = {
+  model: string;
+  options?: GetContentOptions;
+  content?: BuilderContent;
+  children: any;
+  contentLoaded?: (content: BuilderContent) => void;
+};
+
+export function RenderContent(props: RenderContentProps) {
+  const content: BuilderContent | undefined =
+    props.content || useBuilderData(props.model, props.options);
+  const state = useState({
+    get css() {
+      return '';
+    },
+    onWindowMessage(event: MessageEvent) {
+      const message = event.data;
+      if (!message) {
+        return;
+      }
+      switch (message.type) {
+        case 'builder.patchUpdates': {
+          const { data } = message;
+          if (!content) {
+            return;
+          }
+          if (!(data && data.data)) {
+            break;
+          }
+          const patches = data.data[content.data?.id];
+          if (!(patches && patches.length)) {
+            return;
+          }
+
+          if (location.href.includes('builder.debug=true')) {
```

Comment:
```suggestion
          if (location.href.includes('builder.debugger=true')) {
```
The we could leave `builder.debug=true` for `console.log` stuff

Reply:
oops I actually wanna get rid of this